packages/wrangler/templates/new-worker-email.ts

```diff
@@ -0,0 +1,26 @@
+/**
+ * Welcome to Cloudflare Workers! This is your first Email worker.
+ *
+ * Learn more at https://developers.cloudflare.com/email-routing/email-workers/
+ */
+
+export interface Env {
+	// Example binding to KV. Learn more at https://developers.cloudflare.com/workers/runtime-apis/kv/
+	// MY_KV_NAMESPACE: KVNamespace;
+	//
+	// Example binding to Durable Object. Learn more at https://developers.cloudflare.com/workers/runtime-apis/durable-objects/
+	// MY_DURABLE_OBJECT: DurableObjectNamespace;
+	//
+	// Example binding to R2. Learn more at https://developers.cloudflare.com/workers/runtime-apis/r2/
+	// MY_BUCKET: R2Bucket;
+}
```

Comment:
This can be generated with `wrangler types` which creates a DTS file for the user.

Reply:
The point of this mostly commented-out `interface` is to let folks know about other bindings they can use, FWIW

Reply:
Alright, I won't block on this.